scripts/zones/Jugner_Forest/npcs/qm1.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+-- Area: Jugner Forest
+--  NPC: qm1 (???)
+-- Involved in Quest: A Timely Visit
+-- !pos -310 0 407
+-----------------------------------
+ID = require('scripts/zones/Jugner_Forest/IDs')
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    if player:getCharVar("Quest[0][105]Prog") ~= 6 then
```

Comment:
quest lua

Reply:
This is actually the default message. Don't know why I pet this unnecessary check there.